distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Split_Tokenize.enso

```diff
@@ -234,3 +348,25 @@ handle_nothing function = x-> case x of
 ## PRIVATE
    Repeat a computation n times.
 repeat_each n ~action = 0.up_to n . each _-> action
+
+## PRIVATE
+   Name a column by appending an integer to a base column name.
+default_column_namer : Text -> Integer -> Text
+default_column_namer base_name i = base_name + " " + i.to_text
+
+## PRIVATE
+   Pad or truncate a vector to be a specified length; if altered, report
+   it as a Column_Count_Mismatch warning.
+uniform_length : Integer -> Vector Any -> Problem_Builder -> Vector Any
+uniform_length target_length v problem_builder = if v.length == target_length then v else
+    problem = Column_Count_Mismatch.Error target_length v.length
+    problem_builder.report_other_warning problem
+    case v.length < target_length of
+        True -> # Pad.
+            num_paddings = target_length - v.length
+            new_v = Vector.new_builder
+            new_v.append_vector_range v
+            repeat_each num_paddings <| new_v.append Nothing
+            new_v.to_vector
+        False -> # Truncate.
+            v.take target_length
```

Comment:
```suggestion
		# Truncate.
        False -> 
            v.take target_length
```

I think there's this convention that if the comment is to apply to a whole case branch, it should be like the above.

IIRC we have a concept of `##` comments getting attached to expressions, and adding it like suggested will attach it to this case-branch.